src/actionCreators/resources.js

```diff
@@ -78,10 +78,10 @@ export const retrieveResource = (currentUser, uri, errorKey, asNewResource) => (
 }
 
 // A thunk that publishes (saves) a new resource in Trellis
-export const publishResource = (resourceKey, currentUser, group, errorKey) => (dispatch, getState) => {
+export const publishResource = (resourceKey, currentUser, group, errorKey) => async (dispatch, getState) => {
```

Comment:
This may or may not work, and may have side effects because I didn't bother to chase the promises up any farther in the stack.